### ISSUE TITLE
"Overhaul" - Will need to update readme with new CloudFormation stack / IAM policy requirements

### ISSUE THREAD

**Issue by kenthua** [2015-Middleware-Keynote/demo-ansible#111]
Need to update the readme with new policy requirements.

This custom inline policy will work, but is very broad.

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "Stmt1459269951000",
            "Effect": "Allow",
            "Action": [
                "cloudformation:*",
                "iam:*"
            ],
            "Resource": [
                "*"
            ]
        }
    ]
}
```

**Comment by alberttwong** [comment]
I had the same problem.   Needed 2 AWS policies in addition to what Kent wrote.

**Comment by kenthua** [comment]
Testing additional policies right now, will do a PR when it deploys successfully.

**Comment by thoraxe** [comment]
Potentially fixed by #128 once it merges.
